### DISCUSSION TITLE
Recommended MOSFET

### DISCUSSION THREAD

**Discussion by mjparme** [ZeroZeroOne-dev/00Key#4] [Q&A] [answered]
What is the recommended MOSFET for Q1?

**Comment by ZeroZeroOne-dev** [comment, marked as answer]
Hi!

Sorry for the lack of documentation, I really should get on it.
The recommended mosfet is a IRL540N.

Let me know if you have any more questions.

**Reply by mjparme** [reply]
Thanks for the fast response and the info! I assume that resistor right below the MOSFET is a pull-down resistor for the gate. Is 10K a good value for that resistor? Are the other resistors 470 Ohms? 

And one last question, I see you have a diagram for which pins are row/columns on the teensy but do you have a diagram for which keys are in columns with each other and which column goes to which column pin and which row goes to which row pin? With that info a QMK configuration could be created (although I don't mind the custom firmware...seems straightforward enough).

I really like this design, looking forward to building it.

**Reply by ZeroZeroOne-dev** [reply]
10K resistor for the mosfet is correct yet, but I used 560 Ohms for the LED resistors, it still gets very bright, i usually only run them at 20% via the mosfet. 

QMK wasnt supported on the teensy 4.1 yet when I made this keyboard but I know that an [effort was made to get it to work](https://github.com/kinx-project/kint/issues/5). If you do try to get QMK to work on it be sure to let me know I would be very interested 😄.

Here are some screenshots from the PCB layout file that show the row/col connections. 

![image](https://user-images.githubusercontent.com/254635/141369366-496f1e71-a2f2-4e96-b4ec-85f6f6067262.png)
![image](https://user-images.githubusercontent.com/254635/141370214-fd896f53-0f37-4b92-9e3a-69124ba223b7.png)

Glad you like the design, good luck!

(I might do a bit of cleanup in the next few days, I especially need to update the SVG files for the case, those are outdated)

**Reply by mjparme** [reply]
Thanks for the info. I have no idea why I didn't just look at the schematic in kicad in the first place. Sorry for being dumb :-) I'll use your firmware at first and then take a stab at QMK.
